platform/broadcom/sonic-platform-modules-ufispace/s9301-32db/modules/pddf_custom_sysstatus_module.c

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright 2019 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ *
+ * A pddf kernel module for system status registers
+ */
+
+#define __STDC_WANT_LIB_EXT1__ 1
+#include <linux/kernel.h>
+#include <linux/module.h>
+#include <linux/jiffies.h>
+#include <linux/i2c.h>
+#include <linux/hwmon.h>
+#include <linux/hwmon-sysfs.h>
+#include <linux/err.h>
+#include <linux/mutex.h>
+#include <linux/sysfs.h>
+#include <linux/slab.h>
+#include <linux/delay.h>
+#include <linux/dmi.h>
+#include <linux/kobject.h>
+#include "../../../../pddf/i2c/modules/include/pddf_client_defs.h"
+#include "../../../../pddf/i2c/modules/include/pddf_sysstatus_defs.h"
+
+#define _memset(s, c, n) memset(s, c, n)
```

Comment:
Why use `_memset` at all, instead of just using `memset`?

Reply:
A workaround to pass the Semgrep code check. Semgrep recommends using the memset_s() function instead. 
However, there is no memset_s() function in the kernel code. This code will not impact anything.